### ISSUE TITLE
Collapse guides into this repo

### ISSUE THREAD

**Issue by Fryguy** [ManageIQ/manageiq.org#859]
guides are fetched separately, but I think it would just be easier to store them all here, particular since they aren't versioned.  I think we can just simply merge the histories together and drop ManageIQ/guides.

**Comment by chessbyte** [comment]
Documentation in this repo may be mixing concerns.  Additionally, some people do access the guides repo directly on GitHub without going through the ManageIQ.org website.

**Comment by Fryguy** [comment]
Yeah I think they should go through the website...plus we get "proper" urls that way.
